content/800-data-platform/05-data-browser.mdx

```diff
@@ -0,0 +1,142 @@
+---
+title: 'Data Browser'
+metaTitle: 'Data Browser'
+metaDescription: 'Data Browser'
+toc: true
+tocDepth: 3
+---
+
+<TopBlock>
+
+Use the **Data Browser** in the Prisma Data Platform to view and edit your application data.
+
+</TopBlock>
+
+## Models (Tables or Collections)
+
+When you first open the **Data Browser**, you can review the tables in your application database which are based on the models defined in the Prisma schema file.
+
+<img src="https://res.cloudinary.com/prismaio/image/upload/v1657146172/docs/pdp/data-browser-models-view_zs1ihu.png" alt="Data Browser - Models view"/>
+
+You can select a table and its data opens in a new tab.
+
+### Open and close tables
+
+To open another table, click the **+** button.
+
+To close a table, click the the **x** button in the table tab.
+
+<img src="https://res.cloudinary.com/prismaio/image/upload/v1657146172/docs/pdp/data-browser-close-open-table_g7xho7.png" width="375px" alt="Data Browser - Open and close tables"/>
+
+### Filter data
+
+Use the **Filters** menu to filter data in the table by adding conditions.
+
+**Steps**
+
+1. Click **Filters** and select **Add a new filter**.
+2. Configure the filter.
+   1. Select the field by which you want to filter.
+   2. Select a comparison operator.
+      - **equals**
+      - **in**
+      - **notin**
+      - **lt**
+      - **lte**
+      - **gt**
+      - **gte**
+      - **not**
+   3. Type the value you want to use for the filter.<br />
+      **Step result**: The table is filtered immediately after you define the first filter.
+3. To add a new filter, click **Add a new filter** and repeat the steps above.
+4. To remove a filter, click the **x** button on the right.
+   <img src="https://res.cloudinary.com/prismaio/image/upload/v1657146172/docs/pdp/data-browser-filters-actions_grtsrh.png" width="450px" alt="Data Browser - add remove apply filters"/>
+
+**Result**
+
+* The data in the table is filtered based on the conditions you added.
+* The default **None** value in the **Filters** menu changes to the number of filters you added.
+
+### Show and hide columns
+
+You can select which columns to view or hide by using the **Fields** menu.
```

Comment:
Is this about hiding columns or not including fields in the result?
(I again would use the Prisma term "fields")

Reply:
Changed. Thanks for this!